aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,295 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
+
+   ```csharp
+   app.UseAntiforgery();
+   ```
+
+1. If the Blazor Server app was configured to disable prerendering, you can continue to disable prerendering for the updated app. In the `App` component, change the value assigned to the `@rendermode` Razor directive attributes for the `HeadOutlet` and `Routes` components.
+
+   Remove the following from the `HeadOutlet` and `Routes` components and replace them with the following `@rendermode` that disables prerendering:
+
+   ```diff
+   - @rendermode="RenderMode.InteractiveServer"
+   ```
+
+   ```razor
+   @rendermode="new InteractiveServerRenderMode(prerender: false)"
+   ```
+
+   For more information, see <xref:blazor/components/render-modes?view=aspnetcore-8.0&preserve-view=true#prerendering>.
+
+### Convert a hosted Blazor WebAssembly app into a Blazor Web App
+
+Blazor WebAssembly apps are supported in .NET 8 without any code changes. Use the following guidance to convert an ASP.NET Core hosted Blazor WebAssembly app into an equivalent .NET 8 Blazor Web App, which makes all of the [new .NET 8 features](xref:aspnetcore-8#blazor) available.
 
 > [!IMPORTANT]
-> The `@bind:get` and `@bind:set` features are receiving further updates at this time. To take advantage of the latest updates, confirm that you've installed the [latest SDK](https://dotnet.microsoft.com/download/dotnet/7.0).
->
-> Using an event callback parameter (`[Parameter] public EventCallback<string> ValueChanged { get; set; }`) isn't supported. Instead, pass an <xref:System.Action>-returning or <xref:System.Threading.Tasks.Task>-returning method to `@bind:set`.
->
-> For more information, see the following resources:
->
-> * [Blazor `@bind:after` not working on .NET 7 RTM release (dotnet/aspnetcore #44957)](https://github.com/dotnet/aspnetcore/issues/44957)
-> * [`BindGetSetAfter701` sample app (javiercn/BindGetSetAfter701 GitHub repository)](https://github.com/javiercn/BindGetSetAfter701)
+> This section focuses on the minimal changes required to convert a .NET 7 ASP.NET Core hosted Blazor WebAssembly app into a .NET 8 Blazor Web App. To adopt all of the new Blazor Web App conventions, follow the guidance in the [Adopt Blazor Web App conventions](#adopt-blazor-web-app-conventions) section. 
+
+1. Follow the guidance in the first three sections of this article:
+
+   * [Update the .NET SDK version in `global.json`](#update-the-net-sdk-version-in-globaljson)
+   * [Update the target framework](#update-the-target-framework)
+   * [Update package references](#update-package-references)
+
+   > [!IMPORTANT]
+   > Using the preceding guidance, update the `Client`, `Server`, and `Shared` projects of the solution.
+
+1. Move the file content from the `Client/wwwroot/index.html` file to a new `App` component file (`App.razor`) created at the root of the `Server` project.
```

Comment:
Should we clarify here that they should also delete index.html? Or is that clear from "move the file content"?

Reply:
I'll say to delete it to make it clear.